docs/md/modules-external-exome-reprocessing.md

```diff
@@ -129,7 +108,40 @@ particular input cram, WFL will not re-submit that workflow.
     }
     ```
 
-### Exec Workload: `/api/v1/exec`
+### Stop Workload: `POST /api/v1/stop`
+
+Included for compatibility with continuous workloads. 
+
+=== "Request"
+
+    ```bash
+    curl -X POST 'https://gotc-prod-wfl.gotc-prod.broadinstitute.org/api/v1/stop' \
+         -H "Authorization: Bearer $(gcloud auth print-access-token)" \
+         -H 'Content-Type: application/json' \
+         -d '{ "uuid": "1337254e-f7d8-438d-a2b3-a74b199fee3c" }'
+    ```
+
+=== "Response"
+
+    ```json
+    {
+      "creator" : "user@domain",
+      "pipeline" : "ExternalExomeReprocessing",
+      "executor" : "https://cromwell-gotc-auth.gotc-prod.broadinstitute.org",
+      "release" : "ExternalExomeReprocessing_vX.Y.Z",
+      "created" : "YYYY-MM-DDTHH:MM:SSZ",
+      "started" : "YYYY-MM-DDTHH:MM:SSZ",
+      "stopped" : "YYYY-MM-DDTHH:MM:SSZ",
```

Comment:
```suggestion
      "created" : "YYYY-MM-ddTHH:mm:ssZ",
      "started" : "YYYY-MM-ddTHH:mm:ssZ",
      "stopped" : "YYYY-MM-ddTHH:mm:ssZ",
```

according to https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html ?

Reply:
I noticed it's following the existing docs, so it's ok if you leave it here we can fix all of them together later.